include/glow/Graph/Graph.h

```diff
@@ -261,10 +270,78 @@ class Function final : public Named {
 
   const Module *getParent() const { return parent_; }
 
+  /// Logs the node creation with a list of input nodes.
+  void logNodeCreation(Node *newNode, llvm::ArrayRef<Node *> inputs) {
+    if (dumpCompilationLogOpt) {
+      std::string logStr = "";
+      logStr.append("[FULL SCOPE: ");
+      logStr.append(logCtx_.getFullScopeName());
+      logStr.append("]");
+      logStr.append(" --- CREATE { (");
+      logStr.append(newNode->getKindName());
+      logStr.append(", ");
+      logStr.append(newNode->getName());
```

Comment:
Same here, but the output should be (`Name: <nodeName>` instead of just `<nodeName>`.